src/Canvacord.js

```diff
@@ -783,6 +783,24 @@ class Canvacord {
         
         return canvas.toBuffer();
     }
+    
+    /**
+     * Oh no! That's Stupid.
+     * @param {string} message Message
+     * @returns {Promise<Buffer>}
+     */
+    static async ohno(message) {
+        if (!message) throw new Error("Message was not provided!");
+        const canvas = createCanvas(1000, 1000);
+        const ohno = canvas.getContext("2d");
```

Comment:
You might use ctx instead of ohno as you can in the whole code and also avoid using `discordapp` cdn as they might get deleted anytime and the function will become useless.

Reply:
I will change ohno to ctx and that link is temporary as the image will be uploaded to canvacord-assets